app/models/pusher.rb

```diff
@@ -4,34 +4,32 @@ class Pusher
   include TraceTagger
   include SemanticLogger::Loggable
 
-  attr_reader :api_key, :user, :spec, :spec_contents, :message, :code, :rubygem, :body, :version, :version_id, :size
+  attr_reader :api_key, :owner, :spec, :spec_contents, :message, :code, :rubygem, :body, :version, :version_id, :size
 
-  def initialize(api_key, body, remote_ip = "", scoped_rubygem = nil)
-    # this is ugly, but easier than updating all the unit tests, for now
+  def initialize(api_key, body, remote_ip = "")
     case api_key
     when ApiKey
       @api_key = api_key
-      @user = api_key.user
-      raise ArgumentError if scoped_rubygem
-      scoped_rubygem = api_key.rubygem
+      @owner = api_key.owner
+      @scoped_rubygem = api_key.rubygem
     else
-      @user = api_key
+      raise ArgumentError, "api_key must be an ApiKey"
     end
 
     @body = StringIO.new(body.read)
     @size = @body.size
     @remote_ip = remote_ip
-    @scoped_rubygem = scoped_rubygem
   end
 
   def process
-    trace("gemcutter.pusher.process", tags: { "gemcutter.user.id" => user.id }) do
+    trace("gemcutter.pusher.process", tags: { "gemcutter.api_key.owner" => owner.to_gid }) do
```

Comment:
You call this `gemcutter.owner` above. I didn't compare them all, but it would make sense if they were consistent.